forecast/Sine_Forecast.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Just a comment: this ipynb is pretty nice as an example; short, simple, visual. I am in favor of keeping this. I am not so sure about the other ipynb.